### PR TITLE
Misc cleanup

### PR DESCRIPTION
**Pull request by M0shr00m**
- Adjusted namespaces

- Cleaned up some code, implemented some newer C# language features for simplification.

No functional changes, just organization and logic simplification.

### CODE REVIEW

File: Heroes.ReplayParser/Foole.Mpq/BitStream.cs [hunk 1]
```diff
@@ -26,10 +26,11 @@
 // OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
 // WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 //
+
 using System;
 using System.IO;
 
-namespace Foole.Mpq
+namespace Heroes.ReplayParser.Foole.Mpq
```
Comment (barrett777) [inline Heroes.ReplayParser/Foole.Mpq/BitStream.cs]: I'd rather not change the Foole.Mpq files, as they originally came from another package.  Could you revert the cosmetic changes to these Foole.Mpq files?

File: Heroes.ReplayParser/MPQFiles/ReplayMessageEvents.cs [hunk 2]
```diff
@@ -41,23 +41,31 @@ public static void Parse(Replay replay, byte[] buffer)
...
                             {
-                                ChatMessage chatMessage = new ChatMessage();
+                                var chatMessage = new ChatMessage
+                                {
+                                        MessageTarget = (MessageTarget) bitReader.Read(3), Message = Encoding.UTF8.GetString(bitReader.ReadBlobPrecededWithLength(11))
+                                };
 
-                                chatMessage.MessageTarget = (MessageTarget)bitReader.Read(3); // m_recipient (the target)
-                                chatMessage.Message = Encoding.UTF8.GetString(bitReader.ReadBlobPrecededWithLength(11)); // m_string
+                                // m_recipient (the target)
```
Comment (barrett777) [inline Heroes.ReplayParser/MPQFiles/ReplayMessageEvents.cs]: Could you move these comments up to the statements they were originally associated with?

File: Heroes.ReplayParser/MPQFiles/ReplayMessageEvents.cs [hunk 3]
```diff
@@ -41,23 +41,31 @@ public static void Parse(Replay replay, byte[] buffer)
...
-                                PingMessage pingMessage = new PingMessage();
+                                var pingMessage = new PingMessage
+                                {
+                                        MessageTarget = (MessageTarget) bitReader.Read(3),
+                                        XCoordinate = bitReader.ReadInt32() - (-2147483648),
+                                        YCoordinate = bitReader.ReadInt32() - (-2147483648)
+                                };
 
-                                pingMessage.MessageTarget = (MessageTarget)bitReader.Read(3); // m_recipient (the target) 
+                                // m_recipient (the target)
```
Comment (barrett777) [inline Heroes.ReplayParser/MPQFiles/ReplayMessageEvents.cs]: Could you move these comments up to the statements they were originally associated with?

**Review by barrett777 — CHANGES_REQUESTED** [review]

File: Heroes.ReplayParser/Foole.Mpq/BitStream.cs (hunk 1 above)
Comment (M0shr00m) [reply Heroes.ReplayParser/Foole.Mpq/BitStream.cs]: Fixed with the latest commit

**Review by barrett777 — APPROVED** [review]

**Pull request merged**